ts/src/blockchaincom.ts

```diff
@@ -168,6 +168,55 @@ export default class blockchaincom extends Exchange {
                 'apiKey': false,
                 'secret': true,
             },
+            'options': {
+                'networks': {
+                    'ERC20': 'ETH',
+                    'TRC20': 'TRX',
+                    'ALGO': 'ALGO',
+                    'CARDANO': 'ADA',
+                    'ADA': 'ADA',
+                    'AR': 'AR',
+                    'COSMOS': 'ATOM',
+                    'ATOM': 'ATOM',
+                    'AVAX_C': 'AVAX',
+                    'BCH': 'BCH',
+                    'BSV': 'BSV',
+                    'BTC': 'BTC',
+                    // 'BEP20': 'BNB', // todo
+                    'DCR': 'DCR',
+                    'DESO': 'DESO',
+                    'DASH': 'DASH',
+                    'CELO': 'CELO',
+                    'CHZ': 'CHZ',
+                    'POLYGON': 'MATIC',
+                    'MATIC': 'MATIC',
+                    'DOGECOIN': 'DOGE',
```

Comment:
@ttodua I'm a bit confused, aren't we going to keep using the short name for the networks?

Reply:
yes, there are short names in both cases, look at it again:
```
                    'POLYGON': 'MATIC',
                    'MATIC': 'MATIC',
                    'DOGECOIN': 'DOGE',
                    'DOGE': 'DOGE',

```

so, there POLYGON and DOGECOIN are just as aliases (because those two networks are somewhat major and large ones, so I've left them as aliases. later, the 10 (or so) aliases will be moved into base

Reply:
about POLYGON, CARDANO and ATOM, my question was answered by Carlo https://discord.com/channels/690203284119617602/1059793289575923762/1128352891825045567  and I've just considered that as your thought too

Reply:
> so, there POLYGON and DOGECOIN are just as aliases

@ttodua but I don't think this will work well with the `networksById` automatic inversion, let's say you have
```
networks:
   - DOGE : doge
   - DOGECOIN: doge
```

If you try to invert what value will be assigned to the key `doge`?

Reply:
that's a good question, however, I've taken care of it already with the plan which I'm going through.
so, the temporary order I have pushed across all PRs are the following:
```
aliasName: xx
unifiedMajorName: xx
```
so the last one overwrites others before it. at this moment across all individual exchanges, i had inserted the "major" name after aliases, so aliases are overwriten during inversion.
you might say that it's not reliable way and I 100% agree, however, I couldnt find any better solution in-between period before the base modifications/updates are merged (which was initially in my "ALL-IN-ONE" pr, but that become impossible to get reviewed together and that's why I'm making small PRs before that final one. after final PR, these aliases will be removed from here and there will be no need/difference in order)

Reply:
@ttodua Yeah I don't think this is a good solution because we're relying on the order of the keys which is not reliable and very hard to assure, why don't we remove the alias? I don't think they add much value and as we are seeing are causing some issues.

Reply:
that was ofc temporary before Base updates, but oky, I've removed aliases from all individual PRs, now you can revise. I'm now starting PR to add an alias support from base